.github/workflows/int_test.yml

```diff
@@ -164,13 +168,16 @@ jobs:
           cluster-name: ${{ env.CLUSTER_NAME }}
           resource-group: ${{ env.RESOURCE_GROUP }}
           custom-locations-oid: ${{ env.CUSTOM_LOCATIONS_OID }}
-      - name: "Get Keyvault ID"
+      - name: "Tox test environment setup"
         run: |
-          KV_ID=$(az keyvault show -n ${{env.KV_NAME}} -g ${{ env.RESOURCE_GROUP }} -o tsv --query id)
-          echo "KV_ID=$KV_ID" >> $GITHUB_ENV
+          cd ${{ env.EXTENSION_SOURCE_DIRECTORY }}
+          python -m pip install tox
+          tox r -vv -e python-int --notest
+          tox r -vv -e python-init-int --notest
```

Comment:
will try without - I wasn't sure if each tox option will try to create multiple tox folders...

Reply:
A few things here - the `--notest` sets up the tox env by pre-installing dependencies and preparing the test env for a quicker run, and also helps break out test setup failures from actual test run failures.

The `python-int` setup only needs to run if we're not containerizing the tests (because tox will do its own setup inside the container), otherwise it's unused. That said, it doesn't technically hurt to do it (other than wasting time) - which we can kind of mitigate by utilizing the `-p` switch to run tox steps in parallel:

`tox r -vv -e "python-int, python-init-int" --notest`


That said, it's probably still better to break up the tox setup into:

- Tox setup for init (happens always in all scenarios we're running using the tests to init)
- Tox setup for full int tests (`if: ${{matrix.feature == 'default' && !inputs.use-container}}`)